drivers/tripplite_usb.c

```diff
@@ -507,6 +507,10 @@ static void decode_v(const unsigned char *value)
 			  input_voltage_scaled  = 230;
 			  break;
 
+		case 6: input_voltage_nominal = 
```

Comment:
Looking at context around, should this be `6` or `'6'`?

Reply:
I think the majority of the Protocol 3005 responses are binary not ASCII, so `6` seems correct, and matches @mbastiaan 's debug log.